### PR TITLE
20200315: Resolves issues with image upload dialog

### PR DESCRIPTION
**Pull request by cybtachyon**
fix(js)

### CODE REVIEW

File: js/DrupalImageEditor.es6.js [hunk 1]
```diff
@@ -1,179 +1,148 @@
...
-      let media_library_opener_parameters = {
-        field_widget_id: this.urlfield.id
-      };
-      let opener_encoded = encodeURIComponent(JSON.stringify(media_library_opener_parameters));
-      this.urlfield.addEventListener('change', function (e) {
+    if(!this.schema.readOnly && !this.schema.readonly) {
+      this.input = this.theme.getFormInputField('text');
+      this.button = this.getButton(this.path + '-media', 'upload', Drupal.t('Select/Upload Media'));
+      // @todo: Add support for multiple file/image URL editors.
+      let media_library_settings = 'media_library_opener_id=patternkit.opener.jsonlibrary' +
```
Comment (Darcness) [inline js/DrupalImageEditor.es6.js]: This is never re-assigned, should be 'const' rather than 'let'.

File: js/DrupalImageEditor.es6.js [hunk 2]
```diff
@@ -1,179 +1,148 @@
...
-          }
-          uploadButton.removeAttribute("disabled");
-        },
-        updateProgress: (progress) => {
-          if (this.progressBar) {
-            if (progress) {
-              this.theme.updateProgressBar(this.progressBar, progress);
-            }
-            else {
-              this.theme.updateProgressBarUnknown(this.progressBar);
```
Comment (Darcness) [inline js/DrupalImageEditor.es6.js]: You're removing this code, but you don't call this function ever anywhere else.  Do you not need this functionality anymore?

File: js/DrupalImageEditor.es6.js (hunk 1 above)
Comment (cybtachyon) [reply js/DrupalImageEditor.es6.js]: oops.exe

File: js/DrupalImageEditor.es6.js (hunk 2 above)
Comment (cybtachyon) [reply js/DrupalImageEditor.es6.js]: nope, no one likes progress bars

we prefer progress breweries

File: js/DrupalImageEditor.es6.js [hunk 3]
```diff
@@ -1,179 +1,148 @@
...
       });
     }
+
     let description = this.schema.description || '';
+
     this.preview = this.theme.getFormInputDescription(description);
     this.container.appendChild(this.preview);
-    this.control = this.theme.getFormControl(this.label, this.urlfield || this.input, this.preview);
+
+    this.control = this.theme.getFormControl(this.label, this.input||this.input, this.preview);
```
Comment (cybtachyon) [inline js/DrupalImageEditor.es6.js]: don't look at me like this

**Pull request merged**